docs/user/lib_mapper/netutilsparser.md

```diff
@@ -6,7 +6,7 @@
 | cisco_aireos | → | cisco_aireos |
 | cisco_asa | → | cisco_asa |
 | cisco_ios | → | cisco_ios |
-| cisco_iosxr | → | cisco_iosxr |
+| cisco_iosxr | → | cisco_xr |
```

Comment:
isn't it iosxr? https://github.com/networktocode/netutils/blob/e4363774b99539181a02d8d19bdf37d42c03fb70/netutils/config/compliance.py#L16

Reply:
The normalized (netmiko) doesn't match `cisco_iosxr` it matches `cisco_xr`